### ISSUE TITLE
No output for creating a component from git

### ISSUE THREAD

**Issue by containscafeine** [redhat-developer/ocdev#264]
```console
$ ocdev component create nodejs nodejsnew --git https://github.com/kadel/nodejs-ex
```
There is no output for this command, we need to give a success message.

**Comment by kadel** [comment]
this should output build logs (build progress)

**Comment by cdrage** [comment]
@kadel Going to mark this as an MVP since this is high priority / need to be fixed before initial ocdev release.

**Comment by kadel** [comment]
@cdrage there is already PR for his

**Comment by containscafeine** [comment]
Fixed in #301
